src/Resources/public/vendor/admin-lte/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we have to commit all the admin-lte files ?
Can't we declare the dependency in the composer.json or something ?
Could we use https://packagist.org/packages/almasaeed2010/adminlte ?